changelogs/fragments/608-deprecate-lookup-plugins.yml

```
@@ -0,0 +1,4 @@
+---
+trivial:
```

```suggestion
deprecated_features:
```

I think @p3ck is right, this should be `deprecated_features` and not `trivial`.